src/components/provider-show/provider-show.js

```diff
@@ -44,14 +44,14 @@ export default function ProviderShow({
         paneTitle={model.name}
         actionMenuItems={actionMenuItems}
         bodyContent={(
-          <DetailsViewSection label="Provider information">
-            <KeyValue label="Packages selected">
+          <DetailsViewSection label={<FormattedMessage id="ui-eholdings.provider.providerInformation" />}>
```

Comment:
This might throw an error into the console, but https://github.com/folio-org/ui-eholdings/pull/469 should fix it.

Reply:
Thanks @cherewaty for making the change in #469